zproject/computed_settings.py

```diff
@@ -1212,7 +1212,7 @@ def ensure_dict_path(d: dict[str, Any], keys: list[str]) -> None:
 
 # Merge SOCIAL_AUTH_SYNC_CUSTOM_ATTRS_DICT into SOCIAL_AUTH_SYNC_ATTRS_DICT.
 # This is compat code for the original SOCIAL_AUTH_CUSTOM_ATTRS_DICT setting.
-# TODO/compatibility: Remove this for release Zulip 10.0.
+# TODO/compatibility: Remove this for release Zulip 11.0.
```

Comment:
Should we just remove this at this point?

Reply:
oh true, makes sense to remove in `11.x` now.